src/store/nvidia.ts

```diff
@@ -1,7 +1,7 @@
 import {Store} from './store';
 
 export const Nvidia: Store = {
-	cartUrl: '',
+	cartUrl: 'https://www.nvidia.com/en-us/shop/geforce/?page=1&limit=9&locale=en-us&gpu=RTX%203080&gpu_filter=RTX%203090~1,RTX%203080~7,RTX%203070~1,TITAN%20RTX~1,RTX%202080%20Ti~2,RTX%202080%20SUPER~16,RTX%202080~5,RTX%202070%20SUPER~26,RTX%202070~15,RTX%202060%20SUPER~7,RTX%202060~36,GTX%201660%20Ti~29,GTX%201660%20SUPER~4,GTX%201660~11,GTX%201650%20Ti~6,GTX%201650%20SUPER~8,GTX%201650~34',
```

Comment:
Since this isn't actually a cart, I think we should remove it.

Reply:
The potential problem here is that even if Nvidia is detected in stock, it will open the API result on Digital River.  That doesn't really help anyone purchase a card, the previous cartUrl doesn't work any longer so this at least brings you to a page where you can purchase a card.